### PR TITLE
Fix link from https://doi.org/joss.00433 to https://doi.org/10.21105/joss.00433 but the visual stay the same

### PR DESCRIPTION
**Pull request by yasirroni**
Fix link from https://doi.org/joss.00433 to https://doi.org/10.21105/joss.00433 but the visual stay the same.

## Description
Fix link from https://doi.org/joss.00433 to https://doi.org/10.21105/joss.00433 but the visual stay the same.
```
[https://doi.org/joss.00433](https://doi.org/10.21105/joss.00433)
```

## Motivation and Context
Fixing the link when clicked, but not chainging the visual to make it the same as in research gate.

## How Has This Been Tested?
Just click the link.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [] I have read the **CONTRIBUTING** document.
- [] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -262,7 +262,7 @@ constrained support. The package was created with [Cookiecutter] and the
 ## Cite us
 Are you using PySwarms in your project or research? Please cite us!
 
-* Miranda L.J., (2018). PySwarms: a research toolkit for Particle Swarm Optimization in Python. *Journal of Open Source Software*, 3(21), 433, https://doi.org/joss.00433
+* Miranda L.J., (2018). PySwarms: a research toolkit for Particle Swarm Optimization in Python. *Journal of Open Source Software*, 3(21), 433, [https://doi.org/joss.00433](https://doi.org/10.21105/joss.00433)
```
Comment (ljvmiranda921) [inline README.md:265]: You can also put 10.21105 inside the square brackets 👍

Comment (yasirroni) [reply README.md:265]: I need to make new PR? 

*I'm not sure how to collaborate. I'll make one. Sorry, noob here.

**Comment by yasirroni** [comment]
Fix link from https://doi.org/joss.00433 to https://doi.org/10.21105/joss.00433.



## Description
Fix link from https://doi.org/joss.00433 to https://doi.org/10.21105/joss.00433 but the visual stay the same.
```
[https://doi.org/10.21105/joss.00433](https://doi.org/10.21105/joss.00433)
```

## Motivation and Context
Fixing the link when clicked to the published paper.

## How Has This Been Tested?
Just click the link.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:


- [ ] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [] I have read the **CONTRIBUTING** document.
- [] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

**Comment by ljvmiranda921** [comment]
Awesome! Thanks a lot for your contribution, @yasirroni !
@all-contributors please add @yasirroni for docs

**Pull request merged**